### DISCUSSION TITLE
Cow or Horse

### DISCUSSION THREAD

**Discussion by mvchn** [trngl8/cow#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mvchn** [comment]
Незрозуміло зараз, що ми проектуємо, коня чи корову.
Ігор за коня (наче). Я за корову.
Треба вирішити що ми з ними робимо

**Reply by mvchn** [reply]
Ще в нас звідкись з'явився дракон. Сферичний дракон треба пізніше, я не знаю чим його годувати.

**Reply by mvchn** [reply]
Бик свійськи́й, або коро́ва (Bos taurus), — вид ссавців роду Бик родини бикових (Bovidae). Слово «корова» позначає самицю виду, а «свійський бик» або «домашній бик» — самця. Розводять заради м'яса, шкіри та молока. У таких релігіях, як індуїзм та буддизм, цю тварину вважають священною, а знищення корови або заподіяння їй шкоди заслуговує кари. В українській народній традиції корова — це годувальниця та основа добробуту сім'ї.

Розрізняють м'ясні, м'ясо-молочні, молочно-м'ясні та молочні породи корів.

**Reply by mvchn** [reply]
Я хочу собі корову

**Reply by mvchn** [reply]
молочно-м'ясну. м'яса ми з неї не беремо

**Reply by mvchn** [reply]
Корова, або бик. Мені в твітері це писали. 

Думаю, треба саме корова female

**Reply by Bujhm** [reply]
> Ще в нас звідкись з'явився дракон. Сферичний дракон треба пізніше, я не знаю чим його годувати.

А дракона кто предложил и почему? ;-)  
И согласен на корову ) не существенно для меня пока.
А конь был для обозначения абстрактной задачи, с неопределёнными или слабо определёнными требованиями.

**Reply by mvchn** [reply]
Тебе нужен конь или нет? 
а то я уже думал что ты коня хочешь, нашел деньги на коня

**Comment by chulivska** [comment]
Я за Корову
Для натхнення: https://photos.app.goo.gl/qfoyS8x4f193S4yk8

**Comment by mvchn** [comment]
Один мільйонер дуже любив кінні перегони, але постійно робив ставку не на того коня і програвав. Тож він вирішив дізнатися наперед який кінь переможе і найняв біолога, математика й фізика. Він дав їм завдання, багато грошей і рік часу.
За рік біолог повідомив:

- Знаючи родовід кожного коня, успіхи його батьків, як його годували і як лікували, я можу точно сказати який кінь переможе.

Математик сказав:

- Маючи точні статистичні дані попередніх забігів коней, я можу приблизно назвати який з них переможе.

Фізик же сказав:

- Мені потрібно ще десять років, п'ятдесят мільйонів доларів, кілька помічників і лабораторія, але я вже побудував модель руху абсолютно пружного сферичного коня в вакуумі!

**Comment by mvchn** [comment]
Досить цікавий тут діалог.

Я думаю що треба зберегти традицію. Код ми пишемо мовою машини, Issues та Pull requests ми пишемо англійською, тексти в дискусіях якоюсь кириличною мовою.
